deploy-templates/templates/rbac/role_developer.yaml

```diff
@@ -3,19 +3,43 @@ kind: Role
 metadata:
   name: tenant-developer
 rules:
-  - apiGroups:
+  - verbs:
+      - list
+      - get
+      - create
+      - watch
+      - update
+    apiGroups:
       - argoproj.io
     resources:
       - applications
-    verbs:
+  - verbs:
+      - create
       - list
       - get
-      - create
       - watch
-      - update
-  - apiGroups:
+    apiGroups:
       - tekton.dev
     resources:
       - pipelineruns
-    verbs:
-      - create
+      - taskruns
+      - tasks
+      - pipelines
+  - verbs:
+      - list
+      - get
+      - watch
+    apiGroups:
+      - v2.edp.epam.com
+    resources:
+      - cdpipelines
+      - cdstagedeployments
+      - codebasebranches
+      - codebaseimagestreams
+      - codebases
+      - gitservers
+      - jiraissuemetadatas
+      - jiraservers
+      - quicklinks
+      - stages
+      - templates
```

Comment:
Default view role should allow an user to list these resources.
https://github.com/epam/edp-install/blob/master/deploy-templates/templates/role_edp_aggregate_view.yaml